launch_testing/launch_testing/tools/output.py

```diff
@@ -64,7 +65,7 @@ def _filter(output):
     return _filter
 
 
-def expected_output_from_file(path):
+def expected_output_from_file(path, encoding: Optional[str] = None):
```

Comment:
While we are adding in new type annotations, can we add one to path as well?

```suggestion
def expected_output_from_file(path: str, encoding: Optional[str] = None):
```

Reply:
Yes. Updated in https://github.com/ros2/launch/pull/717/commits/e445d13fa35765fbcff33b0271991744791ea61e